project1/templates/user.html

```diff
@@ -0,0 +1,9 @@
+<html>
+    
+    <body>
+      <h1>User home page</h1>  
+        
+        <h1>{{ user }}</h1>
+        
+    </body>
+</html>
```

Comment:
This is a user's home page it seems, so this also requires logout button or link right??